### ISSUE TITLE
Need manual font settings

### ISSUE THREAD

**Issue by fre-ac** [enzo1982/freac#221]
Fre:ac converter is a great tool.But i have a question.Default font in my OS is sans 10.The font in version v1.1.4 is too large and cannot be set to another. In 1.1.Alpha it was fine. It's a good idea to have to manually set the font. Thank you for your interest in this issue.

**Comment by fre-ac** [comment]
see attachments:
![freac-1 1 4 2021_11;55;11](https://user-images.githubusercontent.com/85880630/121893932-d5866600-cd1e-11eb-86df-fa1cb24d2628.png)
![freac-1 1alpha 2021_10;56;44](https://user-images.githubusercontent.com/85880630/121893942-d6b79300-cd1e-11eb-835a-c6b64d4656f8.png)

**Comment by fre-ac** [comment]
see attachments:

![V1 1 4](https://user-images.githubusercontent.com/85880630/121894103-05ce0480-cd1f-11eb-86fc-a4212c649e8c.png)
![v1 1 Alpha](https://user-images.githubusercontent.com/85880630/121894109-0797c800-cd1f-11eb-9c8d-8bfa68b01ddd.png)

**Comment by enzo1982** [comment]
fre:ac uses the GTK font and font size on Linux. Older versions had a fixed font size that usually was smaller than the configured font size. That used to be a common complaint about fre:ac.

You have two options to change the font size in fre:ac:

1. Change the font size for all GTK apps using _gsettings_ by adjusting the `org.gnome.desktop.interface/font-name` property.
E.g. if `gsettings get org.gnome.desktop.interface font-name` displays **"Sans 10"**, try setting it to **"Sans 8"** using `gsettings set org.gnome.desktop.interface font-name 'Sans 8'`.
2. Run fre:ac with the `--scale` argument like this: `freac --scale:1.0`
Setting the scale value to 1.0 will result in the original size like in the alpha releases, which didn't support adjusting to the configured system font size.

**Comment by fre-ac** [comment]
danke für Ihre prompte und professionelle Antwort. Ich möchte nur sagen, dass das manuelle Ändern der Schriftart für viele Benutzer praktisch wäre .

**Comment by enzo1982** [comment]
Ich denke nicht, dass eine Option für die Schriftart in fre:ac hinein gehört. Andere Programme bieten solche Einstellungen auch nicht an.

Ich werde aber Möglichkeiten untersuchen, die Systemschrift auch auf anderen Desktops als Gnome zu erkennen. Das sollte das Problem lösen.

Ich werde mir auch anschauen, warum einige Texte abgeschnitten erscheinen, und versuchen das in einem zukünftigen Update zu beheben.

**Comment by fre-ac** [comment]
Ich habe die neue Version 1.1.5 installiert. Es gibt eine Verbesserung bei der Schriftdarstellung, aber etwas könnte verbessert werden. Siehe Anhang. Vielen Dank. 
![v1 1 5 scale1 1](https://user-images.githubusercontent.com/85880630/126384154-54af3c17-e7cb-4c2b-b63f-7ed7468cb954.png)
![v1 1 5](https://user-images.githubusercontent.com/85880630/126384156-4daf0724-e1c2-4334-b7f4-bee410b8617e.png)

**Comment by enzo1982** [comment]
Ja, das Problem mit den abgeschnittenen Texten konnte ich noch nicht beheben.

Welches OS und welche Schriftart ist das genau?

**Comment by fre-ac** [comment]
Mein Betriebssystem ist Slackware 14 XFCE. Sans 10 Schriftart .
